index.html

```diff
@@ -17,11 +19,23 @@
 <body>
 
  <!-- navigation code here -->
+<nav>
```

Comment:
I would make the header here instead because when search engines scan/scrape the page you want your main heading and title and what the page is about to be in there. Right now you have "Lift off" and the paragraph in the header.
https://www.w3schools.com/tags/tag_header.asp